salt/modules/chocolatey.py

```diff
@@ -222,19 +222,19 @@ def bootstrap(force=False, source=None):
             (
                 "Vista",
                 "x86",
-            ): "http://download.microsoft.com/download/A/7/5/A75BC017-63CE-47D6-8FA4-AFB5C21BAC54/Windows6.0-KB968930-x86.msu",
+            ): "https://download.microsoft.com/download/A/7/5/A75BC017-63CE-47D6-8FA4-AFB5C21BAC54/Windows6.0-KB968930-x86.msu",
```

Comment:
I'm actually receiving a 404 on all of these ps_downlaods urls.

ping @twangboy are these still needed? Do we need to update them with url's that do work?